script.js

```diff
@@ -75,20 +93,28 @@ function showCard(event) {
   }
   if (event.target.classList.contains("-1")) {
     event.target.innerHTML = "!";
+    event.target.style.backgroundColor = "white";
```

Comment:
you could also set all of these instances of `"white"` to a const at the top if you ever wanted to change it you would only have to do it in one place